docs/graphql/manual/guides/deployment/kintohub-one-click.rst

```diff
@@ -0,0 +1,61 @@
+.. meta::
+   :description: Deploy Hasura GraphQL engine in one-click on KintoHub
+   :keywords: hasura, docs, guide, deployment, kintohub, one-click
+
+Hasura GraphQL engine in one-click on KintoHub
+==============================================
+
+.. contents:: Table of contents
+  :backlinks: none
+  :depth: 1
+  :local:
+
+The Hasura GraphQL engine is available in one-click on KintoHub. It is packed with a private `Postgres <https://www.postgresql.org/>`__ instance and secured with a free and automatic HTTPS certificate using `Let's Encrypt <https://letsencrypt.org/>`__.
+
+Quickstart
+----------
+
+1. Deploy your private Hasura and Postgres Database
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Click the button below to deploy your Hasura GraphQL engine and Postgres instances in your private environment. (``Ctrl+Click`` to open in a new tab)
+
+.. image:: https://resources.kintohub.com/deploybutton.png
+   :width: 300px
+   :alt: do_deploy_hasura_button
+   :class: no-shadow
+   :target: https://deploy.kintohub.com/hasura
+
+The deployment will take ~1 minute.
+
+2. Open console
+~~~~~~~~~~~~~~~
+
+Once the deployment is done (green), you can access your Hasura Console by clicking on **Copy Url** and pasting the URL in your browser.
+
+Using migration
+---------------
+
+For advanced use case or if you want to use Hasura migrations, you can use our KintoHub template available on `Github <https://github.com/kintohub/hasura-template>`__ and follow this `tutorial <https://blog.kintohub.com/git-ready-with-hasura-your-team-part-2-7-d0f9617cb8f2>`__.
```

Comment:
What advanced use cases are meant here? If you mean Hasura migrations, the sentence can be `For advanced use cases like using Hasura migrations, ...`.